extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -21,8 +21,15 @@ public function check() {
 	}
 
 	public function run() {
-		// I need to pass $this->sDbStarter to CreateWikiLocalJob::changeStarterContributions
-		$starterDatabase = Starters::getStarterByLanguage( $this->taskContext->getLanguage() );
+		$language = $this->taskContext->getLanguage();
+
+		// TODO Clean up in XW-2393
+		if ( (int) $this->taskContext->getVertical() === \WikiFactoryHub::VERTICAL_ID_TV && $language === 'en' ) {
+			$starterDatabase = Starters::STARTER_EN_TV;
+		} else {
+			// I need to pass $this->sDbStarter to CreateWikiLocalJob::changeStarterContributions
```

Comment:
About passing variable to a method, I think :trollface: 
It was there before, I just moved it. It's not well worded but might give the needed context so I don't want to remove it.